aries-site/src/examples/components/input/MaskedInputExample/MaskedInputExample.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Box } from 'grommet';
+
+import { UsageExample } from '../../../../layouts';
+import {
+  MaskedDateExample,
+  MaskedEmailExample,
+  MaskedTimeExample,
+  MaskedPhoneExample,
+} from '.';
+
+export const MaskedInputExample = () => {
+  return (
+    <UsageExample>
+      <Box gap="medium">
+        <Box direction="row-responsive" gap="large">
+          <MaskedEmailExample />
```

Comment:
For each of these examples, as if I click into it and the drop opens, I can't get it to close unless I fill in the field completely. Seems like when I click outside, the drop should close -- this is how it is for MaskedInput on Storybook for grommet

Reply:
What's interesting here is that the Storybook MaskedInput code placed here creates the same behavior...  The difference is that here the MaskedInput is contained in a `<FormField>`. If I remove the FormField wrapper, the expected behavior returns. So, there is something going on in the interaction between FormField and MaskedInput.

Any ideas?

Reply:
Hmm that's interesting -- seems like it could be a grommet thing. No solution off the top of my head but sounds like something to look into with grommet. Don't think it should hold up this PR, but should be a backlog item.

Reply:
Yes, I believe it is. I will create a Codesandbox example to replicate and then file an issue with Grommet.